src/transformers/image_processing_utils.py

```diff
@@ -476,7 +476,10 @@ def is_valid_size_dict(size_dict):
 
 
 def convert_to_size_dict(
-    size, max_size: Optional[int] = None, default_to_square: bool = True, height_width_order: bool = True
+    size: Optional[int] = None,
```

Comment:
Why add a wrong type annotation here? The size can be an int, a tuple, a list. Best to leave as it was IMO.

Reply:
Reverted in bbcbad1